Bundle/CoreBundle/doc/ManageViews.md

```diff
@@ -0,0 +1,31 @@
+#Manage Views
```

Comment:
I'd appreciate a schema to show different objects with their responsabilities or something more visual.